src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.product__review-box {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-bottom: 24px;
+}
+
+.product__stars {
+  display: flex;
+}
+
+.product__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.product__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.product__star--highlighted {
+  background-image: url(../images/star-active.svg);
```

Comment:
Similarly, verify that the image for the highlighted star is correctly placed in the given path and that the image exists to avoid UI issues.